packages/documentation/src/pages/platform/tools/feature-flags.mdx

```diff
@@ -64,57 +64,16 @@ We've created some infrastructure to make this a little easier to do.
 
 ### Creating a flag and turning it on or off
 
-In src/site/utilities/featureFlags.js, you'll see a list of current flags and their state:
-
-```js
-// Edit this to add new flags
-const featureFlags = {
-  FEATURE1: 'feature1',
-};
-
-// Edit this to turn flags on or off
-const flagsByBuildtype = {
-  // localhost: [],
-  localhost: [featureFlags.FEATURE1],
-  // vagovdev: [flags.FEATURE1],
-  vagovdev: [],
-  // vagovstaging: [flags.FEATURE1],
-  vagovstaging: [],
-  // vagovprod: [flags.FEATURE1],
-  vagovprod: [],
-};
-```
-
-In these two objects, you can create a new feature flag and set which web environments it is turned on in. Whenever you need to change what environment has a particular feature, you would add the flag to the array associate with that build type.
-
-### Using flags in Node code
-
-There are a couple ways to use the flag in Node Javascript code. One way is to use the `applyFeatureFlags` helper in a file that you would like to create a feature flagged variant of:
-
-```js
-const someQuery = 'some query info';
-
-module.exports = someQuery;
-applyFeatureFlags(module);
-```
-
-This helper looks through all of the enabled feature flags and checks to see if there are files with the same name as the current file, but with a feature flag in the file path before the extension.
-
-So, for example, if your file is `healthCarePage.graphql.js` and there's a feature flag called `feature1`, the helper will look for `healthCarePage.graphql.feature1.js`. If `feature1` is turned on, then it will replace the exported module data with the export from the file with the `feature1` suffix.
+When running a new vets-website build, the script will fetch all CMS feature flags from Drupal. This allows the flags to be controlled from within Drupal.
 
-If you want to make a smaller conditional logic change than creating a whole new file, you can import the featureFlags file and check the flags directly:
+Before writing any code to use a new feature flag, it must first be created from within Drupal for all three environments. The feature flags can be found at https://dev.cms.va.gov/admin/config/system/feature_toggle for dev. Staging and production have similar pages.
```

Comment:
You could probably make that url a link, as well as the Staging and production text in the next sentence.